zulipterminal/config/keys.py

```diff
@@ -312,6 +312,11 @@ class KeyBinding(TypedDict):
         'help_text': 'Show/hide stream information & modify settings',
         'key_category': 'stream_list',
     },
+    'TOPIC_INFO': {
+        'keys': ['i'],
+        'help_text': 'Show/hide topic information & modify settings',
```

Comment:
This is only useful for this feature right now, so perhaps we should update this slightly to mention updating the resolve status.

Reply:
Is it alright now?